### PR TITLE
New V2-prerelease 2.2.0 (2.0.1-beta.30)

### PR DESCRIPTION
**Pull request by alrod**

### CODE REVIEW

File: cli-feed-v3.json [hunk 1]
```diff
@@ -104,6 +104,16 @@
       "sha2": "A148DC9C2783D2E7E5E50F59BE2EAEA7948213A30876F41171CF20E9F84754F2",
       "FUNCTIONS_EXTENSION_VERSION": "beta"
     },
+    "2.2.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.13",
+      "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.30/Azure.Functions.Cli.no-runtime.2.0.1-beta.30.zip",
+      "localEntryPoint": "func.exe",
```
Comment (ahmelsayed) [inline cli-feed-v3.json]: the `no-runtime` package doesn't have a `func.exe`. @soninaren  is VS using the `localEntryPoint` now? 

Btw, the self-contained builds are not signed. I expect VS to not like that.

**Comment by paulbatum** [comment]
Hold off on this. @soninaren has to an update to do on friday so that the previous V2 release goes out to VS users. We need to confirm that works (due to the need for self contained build since not all users have 2.1 installed). Then we can proceed with another feed update for the latest release.

File: cli-feed-v3.json (hunk 1 above)
Comment (soninaren) [reply cli-feed-v3.json]: VS is using the localEntry Point with the latest update. https://github.com/Azure/app-service-announcements/issues/118. Let me do a quick test to see if it causes issues with unsigned bits.

**Comment by soninaren** [comment]
@paulbatum this is for OGF testing only. We will not move to prod until Friday.

File: cli-feed-v3.json (hunk 1 above)
Comment (ahmelsayed) [reply cli-feed-v3.json]: thanks @soninaren, @alrod update the `cli` to use the `win-x86` package instead of the `no-runtime` one.

**Comment by paulbatum** [comment]
Got it, I didn't look closely enough.

File: cli-feed-v3.json [hunk 2]
```diff
@@ -104,6 +104,16 @@
       "sha2": "A148DC9C2783D2E7E5E50F59BE2EAEA7948213A30876F41171CF20E9F84754F2",
       "FUNCTIONS_EXTENSION_VERSION": "beta"
     },
+    "2.2.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.13",
+      "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.30/Azure.Functions.Cli.win-x86.2.0.1-beta.30.zip",
+      "localEntryPoint": "func.exe",
+      "itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-beta-10185",
+      "projectTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.AzureFunctions.ProjectTemplates/2.0.0-beta-10185",
```
Comment (soninaren) [inline cli-feed-v3.json]: Need to update the templates
```
"itemTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Azure.Functions.Templates/2.0.0-beta-10202",
"projectTemplates": "https://www.myget.org/F/azure-appservice/api/v2/package/Microsoft.AzureFunctions.ProjectTemplates/2.0.0-beta-10202",
```

File: cli-feed-v3.json (hunk 1 above)
Comment (soninaren) [reply cli-feed-v3.json]: I tested it, looks like unsigned bits are working. We should be good to go.

**Pull request merged**